tests/performance/compare-results.js

```diff
@@ -91,16 +91,31 @@ function formatAsMarkdownTable( rows ) {
 	return result;
 }
 
+/**
+ * Returns a Markdown link to a Git commit on the current GitHub repository.
+ *
+ * For example, turns `a5c3785ed8d6a35868bc169f07e40e889087fd2e`
+ * into [https://github.com/wordpress/wordpress-develop/commit/36fe58a8c64dcc83fc21bddd5fcf054aef4efb27](36fe58a).
+ *
+ * @param {string} sha Commit SHA.
+ * @return string Link
+ */
+function linkToSha(sha) {
+	const repoName = process.env.GITHUB_REPOSITORY || 'wordpress/wordpress-develop';
+
+	return `[https://github.com/${repoName}/commit/${sha}](${sha.slice(0, 7)})`;
```

Comment:
@swissspidy I think this is reversed. The text should be within `[]` and the URL within `()`.

Reply:
Yep, just noticed as well